docs/docs/guides/developer_guides/smart_contracts/profiling_transactions.md

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
do we not need to add this to the main docs reference file so it shows up on the sidebars?

Reply:
Its shows up on the sidebar now under "Aztec.nr" - https://6736ede7171fe516edd8a7ec--aztec-docs-dev.netlify.app/guides